### ISSUE TITLE
How to activate an app without opening a target?

### ISSUE THREAD

**Issue by wearhere** [sindresorhus/open#153]
On OS X, at least (not sure about other platforms), it is possible to use `open` to bring an existing window of an app to the foreground. Example:

```sh
open -a Terminal
```

Is it possible to do this with this project? It seems like `target` is [required](https://github.com/sindresorhus/open/blob/66d3bc74a49202bd980a77486febcb635be283dc/index.js#L22) so I won't be able to mimic `open -a Terminal` exactly. I tried passing an empty target

```js
open('', {
  app: 'Terminal'
}).catch((err) => {
  debug('Could not activate Terminal:', err);
});
```

but this opens a new Terminal window rather than bringing the existing one to the foreground.

**Comment by sindresorhus** [comment]
We could potentially make the `target` argument optional, but first I think you should confirm that not specifying target would bring existing window of an app to the foreground on Windows and Linux too. Because if it only works on macOS, there's not much point adding it here.

**Comment by wearhere** [comment]
> I think you should confirm that not specifying target would bring existing window of an app to the foreground on Windows and Linux too

Agreed, I think that this is a great bar for adding the capability to this project. Unfortunately I don't have access to Windows nor Linux setups. Hopefully someone else might be able and willing to test.
